scripts/foglamp

```diff
@@ -485,7 +485,7 @@ foglamp_status() {
 
                 # Show Tasks
                 foglamp_log "info" "=== FogLAMP tasks:" "outonly" "pretty"
-                ps -ef | grep -o 'python3 -m foglamp.tasks.*' | grep -o 'foglamp.tasks.*' | grep -v 'foglamp.tasks\.\*' || true
+                ps -ef | grep -v 'cpulimit*' | grep -o 'python3 -m foglamp.tasks.*' | grep -o 'foglamp.tasks.*'    | grep -v 'foglamp.tasks\.\*' || true
```

Comment:
Are we sure, we want to invert-match?

Reply:
yes, we want all the rows apart from the ones elated to cpulimit

Reply:
So we shall stop showing stats collector task  in foglamp status on Raspbian now? Would not it be better to `| sed 's/cpulimit -l 40 --//g'` and retain the current behavior!